pynamodb/tests/test_attributes.py

```diff
@@ -519,8 +479,6 @@ def test_boolean_deserialize(self):
         BooleanAttribute.deserialize
         """
         attr = BooleanAttribute()
-        assert attr.deserialize('1') is True
-        assert attr.deserialize('0') is True
         assert attr.deserialize(True) is True
```

Comment:
should we leave the tests of of 1 and 0 too?

Reply:
/shrug i deleted the backwards-compatibility so they won't work anyways

Reply:
this will only be in 4.x right? So that current user should not be affect

Reply:
correct -- this is for the `4.x` release so anyone who has followed along with the migration code we released and done the upgrade should not be affected